src/bin/luarocks

```diff
@@ -33,4 +32,4 @@ local commands = {
    test = "luarocks.cmd.test",
 }
 
-cmd.run_command(description, commands, "luarocks.cmd.external", ...)
+cmd.run_command("luarocks", description, commands, "luarocks.cmd.external", ...)
```

Comment:
Couldn't we keep the old signature here and continue to use `util.this_program()` inside of `run_command`?